### PR TITLE
Qa response

### PR DESCRIPTION
**Pull request by biyeun**
added links where they should be
forms are now functional

@orangejenny pls fast track for deploy today
@kaapstorm

### CODE REVIEW

File: templates/prelogin/_sections/home/lead.html [hunk 1]
```diff
@@ -44,14 +44,14 @@
...
-                    <form class="form-signup" role="form">
+                    <form class="form-signup" role="form" method="get" action="{% url 'register_user' %}">
                         <label class="sr-only"
                                for="signUpEmail">{% trans "Email" %}</label>
                         <div class="row">
                             <div class="col-xs-6 col-sm-7 col-lg-8">
                                 <input type="text"
                                        class="form-control input-xl"
-                                       name="email"
+                                       name="e"
```
Comment (orangejenny) [inline templates/prelogin/_sections/home/lead.html]: why shorten this?

File: forms.py [hunk 2]
```diff
@@ -95,4 +102,17 @@ def __init__(self, *args, **kwargs):
...
+                'contact_form': self,
+            }
+            html_content = render_to_string("prelogin/_email/contact_form_email.html", params)
+            text_content = render_to_string("prelogin/_email/contact_form_email.txt", params)
+            recipient = settings.CONTACT_EMAIL
+            subject = "Contact Dimagi Request from prelogin"
+            send_HTML_email(subject, recipient, html_content, text_content=text_content,
+                            email_from=settings.DEFAULT_FROM_EMAIL)
+        except Exception:
+            logging.error("Couldn't send pre-login contact email. "
```
Comment (orangejenny) [inline forms.py]: Shouldn't this also indicate to the caller that it failed? I see contactUsFormIssues defined, but I don't follow what triggers it.

Comment (biyeun) [inline forms.py]: no it shouldn't. we'll still have the info on our side in the exception log

Comment (biyeun) [inline forms.py]: also, that would add time to the implementation and we want to get a quick iteration of this out the door pronto. this should not hold up a pr.

File: templates/prelogin/_sections/home/lead.html (hunk 1 above)
Comment (biyeun) [inline templates/prelogin/_sections/home/lead.html]: because it's visible in the url. don't let this hold up the pr

File: forms.py (hunk 2 above)
Comment (orangejenny) [inline forms.py]: Is anyone actually looking at the logs?

File: templates/prelogin/_sections/home/lead.html (hunk 1 above)
Comment (czue) [inline templates/prelogin/_sections/home/lead.html]: @biyeun, what about it being in the URL makes "e" better than "email"?

File: forms.py (hunk 2 above)
Comment (czue) [inline forms.py]: also curious about the answer to this. do we even get notified of these?

Comment (kaapstorm) [inline forms.py]: Not a big deal, but in general we should either to catch a more narrow range of exceptions, or log what the exception is so we can tell whether it's something we don't expect and want to handle differently.

**Pull request merged**